src/components/MapView/DateSelector/utils.ts

```diff
@@ -15,6 +15,38 @@ export const months = [
   'Dec',
 ];
 
+/**
+ * Return the closest date from a given list of available dates
```

Comment:
This is great! Happy there's a JSDoc, and the typing of availableDates clearly shows its expecting a return type from `getTime` :)